src/pages/home/RegisterOrganization.vue

```diff
@@ -8,10 +8,14 @@
       <home-actions />
       <div class="grid--main">
         <div class="text-4xl">{{ $t('registerOrg.register_org') }}</div>
-        <div class="text-2xl w-3/4">
+        <div class="text-2xl w-full sm:w-3/4">
```

Comment:
What is your intention here? Because this is the equivalent of the previous style because `sm:w-3/4` means everything `sm` or bigger should be `w-3/4`

Reply:
fixed